wca-regulations.md

```diff
@@ -116,7 +116,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 3a1) Competitors must be ready to submit their puzzles when they are called (see [Regulation 2u](regulations:regulation:2u)).
     - 3a2) Puzzles must be fully operational, such that normal scrambling is possible.
     - 3a3) Polyhedral puzzles must use a colour scheme with one unique colour per face in the solved state. Each puzzle variation must have moves, states, and solutions functionally identical to the original puzzle.
-- 3d) Puzzles must have coloured parts, which must be one of the following: coloured stickers, coloured tiles, coloured plastic, or painted/printed colours. All coloured parts of a puzzle must be made of a similar material.
+- 3d) Puzzles must have coloured parts, which define the colour scheme of the puzzle and must be either coloured stickers, coloured tiles, coloured plastic, or painted/printed colours. All coloured parts of a puzzle must be made of a similar material.
```

Comment:
In the links you quoted, people answered that "either" can be used for more than one option if you want to emphasize the exclusive-or.

[Here](http://www.thefreedictionary.com/either) is an explanation for "more than two":
 When either is used as a conjunction, no paraphrase with any is available, and so either is unexceptionable even when it applies to more than two clauses: Either the union will make a counteroffer or the original bid will be refused by the board or the deal will go ahead as scheduled.

So, I'm not a native speaker either and I think that would not be appropriate for a novel, but to emphasize the exclusive options here it's fine. Someone correct me if I am wrong.